src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -186,16 +181,14 @@ Engine::~Engine() {
 }
 
 static bool streamsSet(const ov::AnyMap& config) {
-    return config.count(InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS) ||
-           config.count(ov::num_streams.name());
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    if (config.count(InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS))
+        return true;
+    OPENVINO_SUPPRESS_DEPRECATED_END
+    return config.count(ov::num_streams.name());
 }
 
 void Engine::apply_performance_hints(ov::AnyMap& config, const std::shared_ptr<ov::Model>& model) const {
```

Comment:
should we apply performance hint using already parsed config? 
otherwise we have parsing in multiple places

Or even - move apply performance hints as Config class method. So, config is parsed and then we apply performance hints